convert/src/main/kotlin/com/github/zeldigas/text2confl/convert/markdown/ConfluenceStorageFormatConverters.kt

```diff
@@ -197,7 +197,7 @@ class ConfluenceNodeRenderer(options: DataHolder) : PhasedNodeRenderer, Attribut
         val resolvedLink = context.resolveLink(LinkType.IMAGE, node.url.unescape(), null, null)
         var linkUrl: String = resolvedLink.url
 
-        if (!node.urlContent.isEmpty) {
+        if (!node.urlContent.isEmpty()) {
```

Comment:
I believe there is no difference in behavior, so it should not be touched..

Reply:
Didn't build if i don't write it this way, if it's just me it's probably an ide settings that i missed or something about kotlin i didn't get (not a kotlin dev by the way)